user/views.py

```diff
@@ -111,11 +113,31 @@ def add_movie(request):
            return redirect('user:movie_list')
     else:
         return render(request, 'add_movie.html')
+    
+@register.filter
+def get_item(dictionary, key):
+    return dictionary.get(key)
 
 @login_required
 def movie_list(request):
     movies = Movies.objects.all()
-    return render(request, 'movie_list.html', {'movies': movies})
+
+    # Fetching similar movies from KNN model
+    predicted_similar_movies = {}
+    for movie in movies:
+        try:
+            similar_movies_df = get_similar_movies(movie.title)
+            similar_movies = []
+            for index, row in similar_movies_df.iterrows():
+                similar_movies.append(row['title'])
```

Comment:
can you call some API that displays the suggested movie picture ?

Reply:
Do you mean the pictures for suggested movies?

Reply:
Yup

Reply:
Is there an existing API in the project for movie images or should I look for one?

Reply:
Right now there are 2 apis being used omdb and tmdb. You can probably use omdb and search with the title. When you make the change add the updated screenshots. I saw your earlier ss right now so thanks for letting me know about that

Reply:
Ok thanks for the clarification, I will add the pictures and integrate the suggestions in the movie_details page.